go.sum

```diff
@@ -648,6 +648,10 @@ github.com/vmware-tanzu/vm-operator-api v0.1.3/go.mod h1:mubK0QMyaA2TbeAmGsu2GVf
 github.com/vmware/govmomi v0.20.3/go.mod h1:URlwyTFZX72RmxtxuaFL2Uj3fD1JTvZdx59bHWk6aFU=
 github.com/vmware/govmomi v0.23.2-0.20201015235820-81318771d0e0 h1:vFYRzFxHXDERRZ3nLuLHEvTP1oZwu+tgSb8XHSki2tU=
```

Comment:
Can you execute `go mod tidy` and clean up go.sum file containing older entries for govmomi.

Reply:
Done.